Squeezebox name

dasos opened the pull request:
**Description:**
Reordered query file

**Related issue (if applicable):** fixes #4019 

If the code does not interact with devices:
- [x] Local tests with `tox` run successfully. **Your PR cannot be merged unless tests pass**

dasos committed:
Squeezebox name fix #4019

dasos commented:
Sorry for breaking it before, but thanks to @molobrakos for spotting.

fabaff commented on homeassistant/components/media_player/squeezebox.py:
```diff
@@ -171,10 +173,12 @@ def get(self, command):
 
 
+# pylint: disable=too-many-instance-attributes
+# pylint: disable=too-many-public-methods
```
No longer needed as those are globally disabled now (#4107).

dasos commented:
@fabaff everything okay?

dasos commented:
@balloob Do I need to do anything to get this merged? (Not super familar with next steps!)

balloob commented:
- address the comment by @fabaff 
 - resolve merge conflicts

dasos commented:
OK. Sorry for all this. My repo got out of sync. Not sure where the spurious stuff came from. Sorry for my ignorance @fabaff - I hope I've addressed your concerns, and that it is ready to merge.

fabaff commented:
Thanks :bird:

fabaff reviewed (approved):
Looks fine to me.
